README.md

```diff
@@ -2,24 +2,250 @@
 
-You will need a LaTeX installation on your system.
-To build simply type `make` and it will generate the standard PDF.
-See documentation in header comment of `pmix-standard.tex`, and the macros provided by `pmix.sty`.
+* Organizations become eligible for elevation to Membership upon having at least one representative attend two quarterly meetings within the prior twelve months. All new Members must either request membership or be nominated by an existing ASC Member, and be approved by the ASC according to the voting rules described below. Membership confers representation on the ASC, thus granting the right to participate in formal votes, along with public acknowledgment for the entity as being a Member of the PMIx Standard ASC on the PMIx web site. Each Member organization is allowed one "seat" on the ASC, to be filled as the Member chooses. The official representative for an organization is responsible for notifying the ASC in advance of an alternate should the representative not be attending a meeting in order to confer voting rights to that individual.
+    * Note: Individuals may also become Members so long as their direct employer is not a current Member of the ASC. For these purposes, a consultant is not considered a direct employee - thus, someone consulting for a current ASC Member is eligible to become an individual member should they choose to do so. Individual members who subsequently join Member organizations will be absorbed into the larger organization's membership. This likewise applies to individuals who are affiliated with an existing Member organization (e.g., as a contractor), but are not direct employees of that organization. Individuals requesting membership that are affiliated with multiple ASC members can be granted individual Member status by vote of the ASC. This is intended to preclude undue influence in voting procedures by an organization with multiple individual members.
+
+* A Member is removed from the ASC (a) upon having failed to have at least one representative attend two quarterly meetings in the prior twelve months, or (b) by vote of the other ASC members per the voting rules defined below. Attendance by a non-voting member of the organization (i.e., someone not officially designated as an alternate for voting purposes) counts towards meeting this requirement.
+
+The ASC meets quarterly, with at least one quarterly meeting each year to be held "in-person" at an agreed-upon location unless the ASC Members vote otherwise. A vote to "virtualize" the in-person meeting can only apply to that given year. Other quarterly meetings may be held by teleconference in accordance with the desires of ASC members. ASC quarterly meetings are conducted according to the following schedule:
+
+* 3rd week of January
+* 3rd week of April
+* 3rd week of July
+* 3rd week of October
```

Comment:
I'd argue that specifying the week  of the meeting (even though it is allowed to change) is too specific. I'd recommend saying something more general like 'once per calendar quarter with a preference for the first month of the quarter' (or with better wording than that)

Reply:
Good point:

> ASC quarterly meetings are conducted once per calendar quarter with a preference for a time during the first month of the quarter. Precise meeting dates are determined by the ASC members in accordance with their schedules, subject to the voting rules described below. Subsequent changes to a given meeting's dates must be determined no later than two quarterly meetings before the one being changed.